lib/rules/indent.js

```diff
@@ -284,15 +284,16 @@ module.exports = function(context) {
     }
 
     /**
-     * Check to see if the node arguments are spread across multiple lines
+     * Check to see if the argument before the callee node is multi-line and
+     * there should only be 1 argument before the callee node
      * @param {ASTNode} node node to check
      * @returns {boolean} True if arguments are multi-line
      */
-    function isCallArgsMultiline(node) {
-        var lastArg = node.arguments.length > 0 ? node.arguments[node.arguments.length - 1] : null;
+    function isArgBeforeCalleeNodeMultiline(node) {
+        var parent = node.parent;
 
-        if (lastArg) {
-            return lastArg.loc.end.line > node.loc.start.line;
+        if (parent.arguments.length >= 2 && parent.arguments[1] === node) {
```

Comment:
Is that too specific? Can't the same thing be applied regardless of the argument count?